### PR TITLE
Fix panic

### PR DESCRIPTION
**Pull request by LukasJenicek**

### CODE REVIEW

**Review by klaidliadon — APPROVED** [review]

**Review by david-littlefarmer — APPROVED** [review]

**Review by VojtechVitek — APPROVED** [review]
LGTM. Thanks for the fix!

File: tests/pgkit_test.go [hunk 1]
```diff
@@ -127,6 +127,23 @@ func TestSugarInsertAndSelectRecords(t *testing.T) {
...
 
+func TestGetOneInTransaction(t *testing.T) {
+	ctx := context.Background()
+	q2 := DB.SQL.Select("*").From("accounts")
+
+	var account Account
+	tx, err := DB.Conn.BeginTx(ctx, pgx.TxOptions{})
+	assert.NoError(t, err)
+
+	defer tx.Rollback(ctx)
```
Comment (c2h5oh) [inline tests/pgkit_test.go:138]: This feels wrong - you are committing 5 lines below this, so you'd be rolling back once outside of tx

Comment (LukasJenicek) [reply tests/pgkit_test.go:138]: You are right :pray: 

https://github.com/goware/pgkit/pull/19

Comment (VojtechVitek) [reply tests/pgkit_test.go:138]: nice catch @c2h5oh :) thanks!

**Pull request merged**